test/Microsoft.AspNet.Identity.EntityFramework6.Test/IdentityResultAssert.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Identity.Test
```

Comment:
One of the identity test packages was no longer published, so I had to copy in this class and the TestLogger.